src/components/details/ResourcesSection.tsx

```diff
@@ -129,7 +129,7 @@ const GitOpsResourcesSection: React.FC<GitOpsResourcesSectionProps> = ({
                 <SplitItem>
                   <Stack style={{ marginRight: 'var(--pf-global--spacer--sm)' }}>
                     {resourcesList.map((eachResource) => (
-                      <StackItem>
+                      <StackItem key={eachResource.name}>
```

Comment:
Why did you use the name as the key here whereas the kind is used in the other two places?

Reply:
Otherwise, LGTM